src/cli/options.rs

```diff
@@ -540,7 +544,7 @@ impl Options {
             Arg::with_name("response")
                 .long("response")
                 .short("r")
-                .help("The location of the RFC8183 Parent Response XML file")
+                .help("The location of the idexchange:: Parent Response XML file")
```

Comment:
You probably didn't mean to change this ;-)

Reply:
nope, fixed